source/MaterialXFormat/XmlExport.h

```diff
@@ -19,6 +19,8 @@ class ExportResolver
 
 using ExportResolverPtr = shared_ptr<ExportResolver>;
 
+using SkipFlattenPredicate = std::function<bool(const FilePath&)>;
```

Comment:
We should just use a general type name as with other predicates. Perhaps `FilePathPredicate` ?
Also it's not documented, and noticed that ExportResolver/Ptr are also not documented.
You can look at NodePredicate as an example.